### PR TITLE
DOMA-4129 Priorities

### PR DESCRIPTION
**Pull request by toplenboren**
Add priority field to server

### CODE REVIEW

File: apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js [hunk 1]
```diff
@@ -76,6 +76,13 @@ const BillingIntegrationOrganizationContext = new GQLListSchema('BillingIntegrat
                 validateInput: validateReport,
             },
         },
+
+        priority: {
+            schemaDoc: 'Priority of this billing integration. 1 is the highest.',
+            type: Integer,
+            isRequired: false,
+            defaultValue: 1,
```
Comment (sitozzz) [inline apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js:84]: What about zero or negative values? May be add field validation?

File: apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const {
 const { generateQuerySortBy } = require('@condo/domains/common/utils/codegeneration/generate.gql')
 const { generateQueryWhereInput } = require('@condo/domains/common/utils/codegeneration/generate.gql')
 const { GQLCustomSchema, find } = require('@condo/keystone/schema')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
```
Comment (Kanol) [inline apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js:21]: reorder imports

File: apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js [hunk 3]
```diff
@@ -130,6 +131,7 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                         services: receipt.services,
                         printableNumber: receipt.printableNumber,
                         serviceConsumer: serviceConsumer,
+                        priority: get(receipt, ['contet', 'priority']),
```
Comment (Kanol) [inline apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js:134]: context

Comment (pahaz) [reply apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js:134]: write a test for that case

File: apps/condo/domains/resident/schema/ServiceConsumer.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ const ServiceConsumer = new GQLListSchema('ServiceConsumer', {
         },
 
         billingIntegrationContext: {
-            schemaDoc: 'Billing integration context, that this serviceConsumer is connected to',
+            schemaDoc: '[DEPRECATED] Billing integration context, that this serviceConsumer is connected to',
```
Comment (pahaz) [inline apps/condo/domains/resident/schema/ServiceConsumer.js:65]: why it's deprecated?

**Pull request closed without merging**